src/node_options.cc

```diff
@@ -532,6 +532,9 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             kAllowedInEnvvar);
   AddOption(
       "--experimental-specifier-resolution", "", NoOp{}, kAllowedInEnvvar);
+  AddOption("--experimental-test-coverage", "", NoOp{}, kAllowedInEnvvar);
```

Comment:
I think you want to edit the existing `AddOption` call for this flag, not duplicate it:

https://github.com/JakobJingleheimer/node/blob/opts/expand-allowlist-node_options/src/node_options.cc#L698

Likewise for the others.

Reply:
gah, thanks